### DISCUSSION TITLE
Guided viewing

### DISCUSSION THREAD

**Discussion by mizaki** [ACBF-Advanced-Comic-Book-Format/ACBF-Viewer#2] [Ideas]
This is the reason (along with text translations) why I wanted to get ACBF going again. I don't know how aware you are of it but there are a couple of versions I've seen:

Amazon does it I believe by zooming in on the panel and "next" takes you to the next panel.
Google has "Bubble Zoom" which means you can tap the bubble to... zoom it.

What I wanted to do was:
Panel focus/zoom.

With the bubbles I think there is a couple of options that should be available:
1. Bubble focus - similar to panel focus/zoom but based on text bubbles. (This currently would be more reliable for auto-detection as frames seems a much harder nut to crack.)
2. Bubble zoom - similar to Google but it's zooms automatically and "next" zooms the next bubble.
3. Font size change - Allow the size of the font to be changed (expanding the bubble area/mask at the same time).

When coming to the end of the page panels or bubbles it would move to the next page etc.

I don't know if this is something you think would be better for adding to the GTK4 version?

**Comment by mizaki** [comment]
Okay, it seems it does do panel guided viewing it's just the interface doesn't tell you in anyway with the zoom levels!

**Comment by GeoRW** [comment]
Yes, it does. That's what frames are defined for. But GTK version is not very smooth. That can be improved in GTK4 maybe. I use GTK viewer mostly for library feature and tracking what I read and rate then.
For reading I use Android Viewer on my tablet where zooming, scrolling, "guided view" is smooth.
